etna/models/tbats.py

```diff
@@ -17,27 +17,50 @@ class _TBATSAdapter(BaseAdapter):
     def __init__(self, model: Estimator):
         self.model = model
         self._fitted_model: Optional[Model] = None
+        self._last_train_timestamp = None
+        self._freq = None
 
     def fit(self, df: pd.DataFrame, regressors: Iterable[str]):
         target = df["target"]
         self._fitted_model = self.model.fit(target)
+        self._last_train_timestamp = df["timestamp"].max()
+        self._freq = pd.infer_freq(df["timestamp"], warn=False)
         return self
 
+    def _determine_num_steps_to_forecast(self, last_test_timestamp: pd.Timestamp) -> int:
```

Comment:
May be we should get this function out of class -- and add it to utils.
I guess something like this logic we have already